api/appsDev.ts

```diff
@@ -0,0 +1,18 @@
+import http from '../http';
+import { PublicApp } from '../types/Apps';
+
+const APPS_DEV_API_PATH = 'apps-dev/external/public/v3';
+
+type FetchPublicAppsForPortalResponse = {
+  results: Array<PublicApp>;
+};
+
+export async function fetchPublicAppsForPortal(
+  portalId: number
+): Promise<Array<PublicApp>> {
+  const resp = await http.get<FetchPublicAppsForPortalResponse>(portalId, {
+    url: `${APPS_DEV_API_PATH}/full/portal`,
+  });
+
+  return resp.results || [];
```

Comment:
Will this obfuscate errors? ([axios errors docs](https://axios-http.com/docs/handling_errors))